test/integration/api/offline-entities.js

```diff
@@ -1126,35 +1126,35 @@ describe('Offline Entities', () => {
           body.currentVersion.data.should.eql({ status: 'checked in' });
           body.currentVersion.label.should.eql('auto generated');
           body.currentVersion.branchId.should.equal(branchId);
+          body.currentVersion.branchBaseVersion.should.equal(1);
           should.not.exist(body.currentVersion.baseVersion);
-          should.not.exist(body.currentVersion.branchBaseVersion); // No base version because this is a create, though maybe this should be here.
           should.not.exist(body.currentVersion.trunkVersion);
         });
 
       backlogCount = await container.oneFirst(sql`select count(*) from entity_submission_backlog`);
       backlogCount.should.equal(0);
 
+      await asAlice.get(`/v1/projects/1/datasets/people/entities/12345678-1234-4123-8234-123456789ddd`)
+        .expect(200)
+        .then(({ body }) => {
+          body.currentVersion.data.should.eql({ status: 'checked in' });
+          should(body.conflict).equal(null); // conflict should be null after update-as-create
+        });
```

Comment:
It looks like there's a similar request above for this entity. Could the code here be consolidated with the code for the request above? For example, the code above is making its own assertion about `currentVersion.data`.